weavetest/tx.go

```diff
@@ -0,0 +1,53 @@
+package weavetest
+
+import "github.com/iov-one/weave"
+
+// Tx represents a weave transaction.
+// Transaction represents a single message that is to be processed within this
+// transaction.
+type Tx struct {
+	// Msg is the message that is to be processed by this transaction.
+	Msg weave.Msg
+	// Err if set is returned by any method call.
+	Err error
+}
+
+var _ weave.Tx = (*Tx)(nil)
+
+func (tx *Tx) GetMsg() (weave.Msg, error) {
+	return tx.Msg, tx.Err
+}
+
+func (tx *Tx) Unmarshal([]byte) error {
```

Comment:
Given Msg has Marshal/Unmarshal, probably easy to add them here as well, right?
If that was useful....

Reply:
Good catch! Addded in https://github.com/iov-one/weave/pull/386/commits/94633ea2804df58b6e31b7e9a42eb55c5a3aec3c